recipes/picaso/meta.yaml

```diff
@@ -17,11 +16,11 @@ build:
 
 requirements:
   host:
-    - python {{ python_min }}
```

Comment:
This should stay. Please address only the lines with review comments.

Reply:
The latest commit removed the lines `{% set python_min = "3.7" %}` in each meta.yaml file, and retained the `{{ python_min }}` statements where relevant (`host`, `run`, and `test` sections.)